docs/src/gui/pyvcp.adoc

```diff
@@ -340,10 +340,10 @@ A label is a way to add text to your panel.
 * '<font>("Helvetica",20)</font>' - specify the font and size of the text.
 * '<relief>FLAT</relief>' - specify the border around the label ('FLAT',
   'RAISED', 'SUNKEN') default is 'FLAT'.
-* '<bd>n</bd>' - where 'n' is the border width when 'RAISED' or 'SUNKEN' borders
+* '<bd>_n_</bd>' - where _n_ is the border width when 'RAISED' or 'SUNKEN' borders
```

Comment:
These changes are all a bit unnecessary as `' '` already makes the whole string italic.

So either everything in italic
(1) `_<bd>n</bd>_` or ` '<bd>n</bd>'`
or everything in monospace
(2) ``` `<bd>n</bd>` ```
or mixed monospace/italic, but that needs to be formatted like this:
(3) `++<bd>++__n__++</bd>++ `

I would tend to (2)

Reply:
```suggestion
* `<bd>__n__</bd>` - where _n_ is the border width when border is `RAISED` or `SUNKEN`.
```

Works in preview here

Reply:
But not with our asciidoc version :(

Reply:
pin names should not be italic imho since italic should be what is variable.  They should quoted or in monospace, preferably not both at the same time.

Reply:
So we all agree on monospace, + emphase (italic) for the variable/placeholder part !?